README.md

```diff
@@ -0,0 +1,162 @@
+```
+attr_filtered = attr.query('dag >= 60')
+attr_filtered
+forcing_temp_renamed_filter = forcing_temp_renamed[forcing_temp_renamed['site_no'].isin(attr_filtered['site_no'])]
+forcing_temp_renamed_filter.reset_index().to_feather('in/forcing/forcing_99.feather', version = 1)
+attr_filtered.reset_index().to_feather('in/attr/attr_99.feather', version = 1)
+```
+
+Now you are almost ready to run the model code.
+
+## Run model
+
+First, set the forcing file name, site attributes name, batch size and hidden layer 
+sizes in StreamTemp-Integ.py on the line numbers shown.  Note that you only specify the base filename here,
+the directory (`in`) is specified elsewhere.
+
+```
+19 forcing_list = ['forcing_99.feather']
+20 attr_list = ['attr_99.feather']
+21              
+22 Batch_list = [ 50]
+23 Hidden_list = [100]
+ ```
+
+Now you can train and make predictions with the model.
+
+From a terminal:
+`python StreamTemp-Integ.py`
+
+Or from an iPython notebook:
+`%run StreamTemp-Integ.py`
+
+## Examine outputs
+Output files will be written to the `output` folder in a subdirectory named according to various model hyperparameters, the start/end dates of the training data, and the random number seed used.  `obs.npy` contains observations for the test time period, and `pred.npy` contains the test period predictions.  The model script is set up to run train the model for six different random number seeds (for size different random neural network weight initializations).
+
+#average together initializations
+```
+#Change these file names to the appropriate ones
+initialization_preds = ['output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_1/All-2010-2016/pred.npy',
+                       'output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_2/All-2010-2016/pred.npy',
+                       'output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_3/All-2010-2016/pred.npy',
+                       'output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_4/All-2010-2016/pred.npy',
+                       'output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_5/All-2010-2016/pred.npy',
+                       'output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_6/All-2010-2016/pred.npy']
+                       
+def load_initializations(files):
+    init_array = []
+    for i in range(len(files)):
+        print(files[i])
+        init_array.append(np.load(files[i])[:,:,0])
+    return(init_array)
+        
+init_array = load_initializations(initialization_preds)
+init_means = np.mean(init_array, axis = 0) 
+```
+
+
+#to compare to observations
+```
+#load in observations, dropping degenerate third dimension
+obs = np.load('output/epochs2000_batch47_rho365_hiddensize100_Tstart20101001_Tend20141001_1/All-2010-2016/pred.npy')[:,:,0]
```

Comment:
Seems weird that `obs` gets loaded from a file called `pred.npy`...what's happening here?